### ISSUE TITLE
Create YAML file for term Category configuration

### ISSUE THREAD

**Issue by twhetzel** [SciCrunch/SciGraph#48]
Creating to link to Jira Issue: https://support.crbs.ucsd.edu/browse/NIF-13553

**Comment by ccondit** [comment]
Reloaded

**Comment by ccondit** [comment]
That's because that IRI (http://purl.obolibrary.org/obo/WBbt_0007833) doesn't appear in the import chain. Should it?

**Comment by twhetzel** [comment]
No, it is not in the import chain. I originally thought that all the terms in the list to use for the yaml file were already confirmed to be in the import chain. Seems best to wait for Maryann to provide the list of categories to use for NIFSTD and move forward from there.

**Comment by twhetzel** [comment]
I'll email the file since I can't attach a .yaml file, but here are the categories:
curies:
  http://purl.obolibrary.org/obo/OBI_0100026 : organism
  http://purl.obolibrary.org/obo/DOID_4 : disease
  http://purl.obolibrary.org/obo/UBERON_0001062 : anatomical entity
  http://purl.obolibrary.org/obo/CHEBI_23367 : molecular entity
  http://purl.obolibrary.org/obo/chebi.owl#CHEBI_23367 : molecular entity
  http://purl.obolibrary.org/obo/ERO_0000007 : technique
  http://purl.bioontology.org/NEMO/ontology/NEMO.owl#NEMO_9559000 : cell
  http://purl.obolibrary.org/obo/GO_0008150 : biological process
  http://purl.obolibrary.org/obo/GO_0005575 : subcellular entity
  http://purl.obolibrary.org/obo/PATO_0000001 : quality

**Comment by twhetzel** [comment]
Other cell terms displayed in Protege looked deprecated. Here is "cell" from NIF-Cell:
http://ontology.neuinfo.org/NIF/BiomaterialEntities/NIF-Cell.owl#sao1813327414

**Comment by cmungall** [comment]
CL:0000000 ! cell
which is declared equivalent to
 GO:0005623 ! cell 
which is declared equivalent to
  NIFSTD:sao1813327414

The CL one may eventually be obsoleted and replaced_by the GO one. I think the SAO one should be gone from the import chain, as it';s replaced the the CC subset of GO

In the interim, I believe SG will make use of equivalence axioms in finding category

TL;DR - use the GO one

**Comment by twhetzel** [comment]
Sure, just saw the YAML file here [1] needs to be updated in the categories section to:
- Remove http://purl.bioontology.org/NEMO/ontology/NEMO.owl#NEMO_9559000 : cell
and Add http://purl.obolibrary.org/obo/GO_0005623 

@ccondit is a change I can make and then re-load nifstd?

[1] https://bamboo.crbs.ucsd.edu/build/admin/edit/editBuildTasks.action?buildKey=NIF-NIFSCIGRAPH-JOB1

**Comment by ccondit** [comment]
I updated the job and the job is running. @twhetzel - feel free to update this as necessary. Just note that you'll need to synchronize this between development, stage, and production by hand.

**Comment by twhetzel** [comment]
Super! @ccondit I'll need a short demo on these tasks
